<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioVideoPlaybar</class>
 <widget class="QWidget" name="AudioVideoPlaybar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>58</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox {
    spacing: 5px;
}

QCheckBox::indicator {
    width: 26px;
    height: 26px;
}

QCheckBox::indicator:unchecked {
    image: url( :/images/HardCodedIcons/repeat_off.png);
}

QCheckBox::indicator:checked {
    image: url( :/images/HardCodedIcons/repeat_on.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="14">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="lcdTime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="seekBarLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbTotalTime">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0:00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="audioOnlyButton">
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>play audio only without video</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:checked {
    background-color: red;
}

QPushButton:!checked {
    background-color: green;
}

QPushButton:!enabled {
    background-color: #AAAAAA;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QCheckBox" name="fadeInCheckbox">
     <property name="toolTip">
      <string>fade in when play</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    spacing: 5px;
}

QCheckBox::indicator {
    width: 26px;
    height: 26px;
}

QCheckBox::indicator:unchecked {
    image: url( :/images/HardCodedIcons/fadeIn_off.png);
}

QCheckBox::indicator:checked {
    image: url( :/images/HardCodedIcons/fadeIn_trigger.png);
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QPushButton" name="frameForwardButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
 background: transparent;
}

QPushButton:pressed{
 background: rgb(200,200,200);
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QCheckBox" name="subtitleCheckBox">
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    spacing: 5px;
}

QCheckBox::indicator {
    width: 26px;
    height: 26px;
}

QCheckBox::indicator:unchecked {
    image: url( :/images/HardCodedIcons/subtitle_off.png);
}

QCheckBox::indicator:checked {
    image: url( :/images/HardCodedIcons/subtitle_on.png);
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="stopButton">
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="11">
    <widget class="QCheckBox" name="repeatCheckBox">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="playButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="rewindButton">
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="frameBackwordButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
 background: transparent;
}

QPushButton:pressed{
 background: rgb(200,200,200);
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="12">
    <widget class="QPushButton" name="secondaryButtons">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">|</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>2</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pauseButton">
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>11</width>
       <height>11</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
